README.md

```diff
@@ -1,5 +1,7 @@
 <img src="https://cdn.weareasterisk.com/product-assets/gavel/banner.png" width="450" height="150" alt="Gavel banner">
 
-**Gavel** is a project expo judging system. Documentation can be found at the link below.
+**Gavel** is a project expo judging system.
 
-# [Gavel Documentation](https://gavel.weareasterisk.com/)
\ No newline at end of file
+[Click here to create a Heroku instance](https://dashboard.heroku.com/new?template=https%3A%2F%2Fgithub.com%2Ftamuhack-org%2Fgavel2%2Ftree%2Fmaster)
```

Comment:
Can you make this link to this repo instead of to the tamu one?

Reply:
Done.